config/collector_definitions.yaml

```diff
@@ -21,12 +21,12 @@
 ---
 openshift:
   default:
-    image: "topological-inventory-ci/topological-inventory-collector-openshift:latest"
+    image: "docker-registry.default.svc:5000/topological-inventory-ci/topological-inventory-collector-openshift:latest"
```

Comment:
I'm not sure why this change was needed. I think this should probably be removed before we merge.

Also, I'm not sure where we will actually get the image from. I doubt it will be in the `topological-inventory-ci` namespace. Should we publish those on dockerhub or something?

Reply:
The old address didn't work for me in the test environment.

> Also, I'm not sure where we will actually get the image from. I doubt it will be in the topological-inventory-ci namespace. Should we publish those on dockerhub or something?

I like the idea of it being published somewhere, but I think we won't be pulling from dockerhub in production.

Reply:
Agreed, I'm just having trouble coming up with a sane default value.